src/sql/workbench/contrib/queryplan2/browser/queryPlan.ts

```diff
@@ -281,7 +283,7 @@ export class QueryPlan2 implements ISashLayoutProvider {
 	}
 
 	private populate(node: InternalExecutionPlanNode, diagramNode: any): any {
-		diagramNode.label = node.name;
+		diagramNode.label = node.subtext.join(this.textResourcePropertiesService.getEOL(undefined));
```

Comment:
So the subtext also includes the name already then?

Reply:
Yeah, the subtext contains both the name and the cost for that operation.